### ISSUE TITLE
cannot install this on Ubuntu: error: command 'g++' failed with exit status 1

### ISSUE THREAD

**Issue by cnlinxi** [JeremyCCHsu/Python-Wrapper-for-World-Vocoder#43]
When I install this pyworld with `pip install pyworld` or git clone this repo, I get error: command 'g++' failed with exit status 1. 

g++ -pthread -shared -B /home/yons/anaconda3/compiler_compat -L/home/yons/anarld.o build/temp.linux-x86_64-3.6/lib/World/src/dio.o build/temp.linux-x86_64-3.6o build/temp.linux-x86_64-3.6/lib/World/src/synthesis.o build/temp.linux-x86_64-3World/src/synthesisrealtime.o build/temp.linux-x86_64-3.6/lib/World/src/codec.o b6_64-3.6/pyworld/pyworld.cpython-36m-x86_64-linux-gnu.so
anaconda3/compiler_compat/ld: cannot find -lm
anaconda3/compiler_compat/ld: cannot find -lpthread
anaconda3/compiler_compat/ld: cannot find -lc
collect2: Error： ld return 1
error: command 'g++' failed with exit status 1

I have no idea to do this. Thanks for your help.

**Comment by nicolaspanel** [comment]
same issue here, but installing from git repo works: `pip install git+https://github.com/JeremyCCHsu/Python-Wrapper-for-World-Vocoder.git`

**Comment by nicolaspanel** [comment]
@JeremyCCHsu here is the full error
```
  running build_ext
  building 'pyworld.pyworld' extension
  creating build/temp.linux-x86_64-3.6
  creating build/temp.linux-x86_64-3.6/pyworld
  creating build/temp.linux-x86_64-3.6/lib
  creating build/temp.linux-x86_64-3.6/lib/World
  creating build/temp.linux-x86_64-3.6/lib/World/src
  x86_64-linux-gnu-gcc -pthread -DNDEBUG -g -fwrapv -O2 -Wall -g -fstack-protector-strong -Wformat -Werror=format-security -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -Ilib/World/src -I/usr/include/python3.6m -I/.venvs/callity-M_MHJR5O/include/python3.6m -I/.venvs/callity-M_MHJR5O/lib/python3.6/site-packages/numpy/core/include -c pyworld/pyworld.cpp -o build/temp.linux-x86_64-3.6/pyworld/pyworld.o
  x86_64-linux-gnu-gcc: error: pyworld/pyworld.cpp: No such file or directory
  x86_64-linux-gnu-gcc: fatal error: no input files
  compilation terminated.
  error: command 'x86_64-linux-gnu-gcc' failed with exit status 1
  ----------------------------------------
  ERROR: Failed building wheel for pyworld
```

**Comment by JeremyCCHsu** [comment]
@nicolaspanel @cnlinxi 
Thank you for reporting this issue.
`pip install pyworld` continued on my machine even after the error. 😲

May you provide more info about your OS and packages?

Here's mine:
```shell
lsb_release -a
# No LSB modules are available.
# Distributor ID: Ubuntu
# Description:    Ubuntu 18.04.1 LTS
# Release:        18.04
# Codename:       bionic

gcc -v
# gcc version 7.4.0 (Ubuntu 7.4.0-1ubuntu1~18.04.1)

pip -V
# pip 19.2.3 from /home/jrm/miniconda3/envs/pyw/lib/python3.6/site-packages/pip (python 3.6)
```

BTW, my machine is a Surface Pro 4 running Windows Subsystem Linux (WSL).

**Comment by JeremyCCHsu** [comment]
resolved in #52
